src/name.rs

```diff
@@ -617,6 +617,30 @@ impl NamespaceResolver {
             None => ResolveResult::Unbound,
         }
     }
+
+    #[inline]
+    pub fn iter(&self) -> impl Iterator<Item = (PrefixDeclaration, Namespace)> {
+        self.bindings[2..].iter().enumerate().filter_map(|(i, n)| {
+            // We check if the key has not been overriden
+            if self.bindings[i + 3..]
```

Comment:
Could you add comments here that explains what the magic numbers 2 and 3 means

Reply:
Done!